src/tools/runk/README.md

```diff
@@ -0,0 +1,277 @@
+# runk
+
+## Overview
+
+`runk` is a standard OCI container runtime written in Rust based on a modified version of
+the [Kata Container agent](https://github.com/kata-containers/kata-containers/tree/main/src/agent), `kata-agent`.
+
+`runk` conforms to the [OCI Container Runtime specifications](https://github.com/opencontainers/runtime-spec).
+
+Unlike the [Kata Container runtime](https://github.com/kata-containers/kata-containers/tree/main/src/agent#features),
+`kata-runtime`, `runk` spawns and runs containers on the host machine directly.
+The user can run `runk` in the same way as the existing container runtimes such as `runc`,
+the most used implementation of the OCI runtime specs.
+
+## Why does `runk` exist?
+
+The `kata-agent` is a process running inside a virtual machine (VM) as a supervisor for managing containers
+and processes running within those containers.
+In other words, the `kata-agent` is a kind of "low-level" container runtime inside VM because the agent spawns and runs containers according to the OCI runtime specs.
+However, the `kata-agent` does not have the OCI Command-Line Interface (CLI) that is defined in the [runtime spec](https://github.com/opencontainers/runtime-spec/blob/master/runtime.md).
+The `kata-runtime` provides the CLI part of the Kata Containers runtime component,
+but the `kata-runtime` is a container runtime for creating hardware-virtualized containers running on the host.
+
+`runk` is a Rust-based standard OCI container runtime that manages normal containers,
+not hardware-virtualized containers.
+`runk` aims to become one of the alternatives to existing OCI compliant container runtimes.
+The `kata-agent` has most of the [features](https://github.com/kata-containers/kata-containers/tree/main/src/agent#features)
+needed for the container runtime and delivers high performance with a low memory footprint owing to the
+implementation by Rust language.
+Therefore, `runk` leverages the mechanism of the `kata-agent` to avoid reinventing the wheel.
+
+## Performance
+
+`runk` is faster than `runc` and has a lower memory footprint.
+
+This table shows the average of the elapsed time and the memory footprint (maximum resident set size)
+for running sequentially 500 containers, the containers run `/bin/true`
+with [detached mode](https://github.com/opencontainers/runc/blob/master/docs/terminals.md#detached) `runk`
+always runs containers with detached mode currently:
+
+|                       | runk (+ oci-kata-agent) | runc  | crun  |
+|-----------------------|-------------------------|-------|-------|
+| time [msec]           | 39.01                   | 53.09 | 35.35 |
+| memory footprint [MB] | 4.223                   | 16.54 | 3.157 |
+
+
+## Status of runk
+
+`runk` is currently an experimental tool.
```

Comment:
I would consider moving this section to the top of the file and making it a ["warning"](https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#warnings-and-other-admonitions). Something like:

> **Warning:** `runk` is currently an experimental tool.

Reply:
I fixed it.